packages/core/src/api-wrappers/defaultApis.ts

```diff
@@ -214,6 +214,11 @@ export const defaultApis = [
           title: 'Your Identity Provider',
           icon: OAuth2Icon,
         },
+        defaultScopes: [
```

Comment:
Tbh I think it'd be better to skip this change and instead document how to apply it to your own app.

It's done by adding a factory to `packages/app/src/apis.ts`:

```ts
  createApiFactory({
    api: oidcAuthApiRef,
    deps: {
      discoveryApi: discoveryApiRef,
      oauthRequestApi: oauthRequestApiRef,
      configApi: configApiRef,
    },
    factory: ({ discoveryApi, oauthRequestApi, configApi }) =>
      OAuth2.create({
        discoveryApi,
        oauthRequestApi,
        provider: {
          id: 'oidc',
          title: 'Your Identity Provider',
          icon: OAuth2Icon,
        },
        defaultScopes: [
          'openid',
          'offline_access',
          'email',
        ],
        environment: configApi.getOptionalString('auth.environment'),
      }),
  }),
```

Reply:
Makes sense, I will do that. Thoughts on leaving `openid` in here at a minimum? I'm not an expert on OIDC but it makes sense to me that there will not be any users who would omit that scope. Example here: https://github.com/dexidp/dex/blob/master/server/oauth2.go#L504

Reply:
Tbh I'd rather skip that one too as these will end up being refactored to not include API references in the core API or default handlers in the auth-backend, so would rather have apps migrate over to specifying their own API instances.

Been raised in some other issues but will mention again that the oidc and oauth API refs in particular don't make much sense, they are useless for plugins since they don't know what they're actually connected to.